lms/djangoapps/ccx/tests/test_field_override_performance.py

```diff
@@ -194,7 +195,8 @@ def instrument_course_progress_render(
         XBLOCK_FIELD_DATA_WRAPPERS=[],
         MODULESTORE_FIELD_OVERRIDE_PROVIDERS=[],
     )
-    def test_field_overrides(self, overrides, course_width, enable_ccx, view_as_ccx):
+    @mock.patch.object(CONTENT_TYPE_GATING_FLAG, 'is_enabled', return_value=True)
```

Comment:
@cpennington not sure what to do here
is_enabled for a waffle flag depends on the request. this method calls a function which doesn't include the request.  
I tried overriding the function on the object directly but this didn't seem to work?

Reply:
I think we should probably use `crum.get_current_request` to get the request in the live code... Which seems to me like we should patch that method to return a generated request (django has a request factory that can create an actual request object, or we could just use a mock).

Reply:
Also, why isn't `override_waffle_flag` enough for this call, when it is for other tests?

Reply:
My understanding is that _is_enabled is the standard wrapper for waffle flags?
It will call crum.get_current_request for this code path  
https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/waffle_utils/__init__.py#L282  
but it also handles various other cases